databallpy/data_parsers/event_data_parsers/statsbomb_parser.py

```diff
@@ -0,0 +1,507 @@
+
+    Returns:
+        MetaData: all metadata information of the current match
+    """
+    with open(match_loc, "r", encoding="utf-8") as f:
+        match_json = json.load(f)[0]
+
+    with open(lineup_loc, "r", encoding="utf-8") as f:
+        lineup_json= json.load(f)
+    
+    home_index = 0 if lineup_json[0]["team_id"] == match_json["home_team"]["home_team_id"] else 1
+    away_index = 1 if home_index == 0 else 0
+
+    home_players = _get_player_info(lineup_json[home_index]["lineup"])
+    away_players = _get_player_info(lineup_json[away_index]["lineup"])
+
+    periods = {
+        "period_id": [1, 2, 3, 4, 5],
+        "start_datetime_ed": [],
+        "end_datetime_ed": [],
+    }
+    match_start = pd.to_datetime(match_json["match_date"]+" "+match_json["kick_off"], utc=True)
+    for id in range(5):
+        if id == 0:
+            periods["start_datetime_ed"].append(match_start)
+            periods["end_datetime_ed"].append(match_start + pd.to_timedelta(45, unit="minutes"))
+        elif id == 1:
+            periods["start_datetime_ed"].append(match_start+ pd.to_timedelta(60, unit="minutes"))
+            periods["end_datetime_ed"].append(match_start + pd.to_timedelta(105, unit="minutes"))
+        else:
+            periods["start_datetime_ed"].append(pd.NaT)
+            periods["end_datetime_ed"].append(pd.NaT)
+
+    metadata = Metadata(
+        match_id=match_json["match_id"],
+        pitch_dimensions=pitch_dimensions,
+        periods_frames=pd.DataFrame(periods),
+        frame_rate=MISSING_INT,
+        home_team_id=match_json["home_team"]["home_team_id"],
+        home_team_name=match_json["home_team"]["home_team_name"],
+        home_players=home_players,
+        home_score=match_json["home_score"],
+        home_formation="",
+        away_team_id=match_json["away_team"]["away_team_id"],
+        away_team_name=match_json["away_team"]["away_team_name"],
+        away_players=away_players,
+        away_score=match_json["away_score"],
+        away_formation="",
+        country=match_json["competition"]["country_name"],
+    )
+    return metadata
+
+
+def _get_player_info(players_data: list) -> pd.DataFrame:
+    """Function to loop over all players and save data in a pd.DataFrame
+
+    Args:
+        players_data (list): for every player a dictionary with info about the player
+
+    Returns:
+        pd.DataFrame: all information of the players
+    """
+    result_dict = {
+        "id": [],
+        "full_name": [],
+        "formation_place": [],
+        "position": [],
+        "starter": [],
+        "shirt_num": [],
+    }
+
+    for player in players_data:
+
+        if len(player["positions"])>0:
+            if player["positions"][0]["from"] == "00:00":
+                starter = True
+            else:
+                starter = False
+            formation_place = player["positions"][0]["position_id"]
+            position = player["positions"][0]["position"].lower()
```

Comment:
Can you make a mapping for all positions that they become one of `["defender", "goalkeeper", "midfielder", "forward"]`? that is used in `match.get_column_ids()`. I think it is also in `databallpy.utils.constants DATABALLPY_POSITIONS`.